### PR TITLE
Add navigation bar and header to walkthrough template.

### PR DESCRIPTION
**Pull request by indhu-gunda**
I completed the walkthrough and added extra information. Also, I decided to use bootstrap to make the page look prettier and started the process with a navigation bar and header.

### CODE REVIEW

File: portfolio/src/main/webapp/css/main.css [hunk 1]
```diff
@@ -0,0 +1,60 @@
+.navbar{background: #6A788D;}
```
Comment (BearBurg) [inline portfolio/src/main/webapp/css/main.css]: Can you reformat it and the following rules?

.example {
  color: #e5e5e5;;
}

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,76 @@
...
+              <a class="nav-link" data-value="blog" href="#">Blog</a>
+            </li>
+
+          </ul>
+      </div>
+    </nav>
+
+    <!-- header -->
+    <header class="header">
+      <div class="overlay">
```
Comment (BearBurg) [inline portfolio/src/main/webapp/index.html]: Do we want this tree structure? Is it possible to have three classes in one div tag?

File: portfolio/src/main/webapp/js/main.js [hunk 3]
```diff
@@ -11,18 +11,21 @@
...
 /**
- * Adds a random greeting to the page.
+ * Adds a random fact to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+function addRandomFact() {
+  const facts =
+      ['My favorite dessert is ricotta cheesecake.', 'It is on my bucket list to see the Aurora Borelis.', 'My favorite place that I have visited is Switzerland'];
```
Comment (BearBurg) [inline portfolio/src/main/webapp/js/main.js]: nit: can we break this line into multiple lines for better readability?

**Review by BearBurg — APPROVED** [review]
LGTM except for some nits.

File: portfolio/src/main/webapp/css/main.css (hunk 1 above)
Comment (indhu-gunda) [reply portfolio/src/main/webapp/css/main.css]: fixed!

File: portfolio/src/main/webapp/js/main.js (hunk 3 above)
Comment (indhu-gunda) [reply portfolio/src/main/webapp/js/main.js]: fixed!

File: portfolio/src/main/webapp/index.html (hunk 2 above)
Comment (indhu-gunda) [reply portfolio/src/main/webapp/index.html]: fixed! I didn't know you could do that.

**Pull request merged**